virustotal_yara/komand_virustotal_yara/actions/scan_file/action.py

```diff
@@ -80,11 +80,3 @@ def matches(built_rules, built_files):
 
         # Returning a array of objects
         return {"results": yara_results}
-
-    def test(self):
```

Comment:
Add a test connection if you are taking it away in the action.

Reply:
Not sure of we want to keep it. There are no connection inputs and all the test did was see if the yara module loads.